jazz-ui/src/app/app.module.ts

```diff
@@ -37,7 +39,7 @@ else{
     AppComponent,
     LandingComponent,
     Error404Component,
-    TestApiComponent,
+    TestApiComponent
```

Comment:
Instead of Directly declaring it in the main app module, you can declare it in shared modules and export the component -  
and declare it only on modules that would use the component.

Reply:
removed and added to environments module

Reply:
I can still find TestAPICOmponent in in app module declerations ...  
you can move it to environment module

Reply:
why is the testAPCOmponent still in app module?
you can declare it in env module if this component is used only in environment module